content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -356,6 +356,7 @@ different Kubernetes components.
 | `VolumeSubpathEnvExpansion` | `false` | Alpha | 1.14 | 1.14 |
 | `VolumeSubpathEnvExpansion` | `true` | Beta | 1.15 | 1.16 |
 | `VolumeSubpathEnvExpansion` | `true` | GA | 1.17 | - |
+| `VolumeCapacityPriority` | `false` | Alpha | 1.21 | |
```

Comment:
Guess you need to make it ordered alphabetically.

Reply:
fixed